docs/source/_toctree_new.yml

```diff
@@ -22,7 +22,7 @@
   title: "Pipelines for Inference"
 - sections:
   - local: optimization/fp16
-    title: "Torch Float16"
```

Comment:
@pcuenca - sorry this file was a bad copy-paste and has been removed on master. Could you rebase your PR here? :-)